lib/js-file.js

```diff
@@ -78,22 +78,44 @@ JsFile.prototype = {
         this._disabledRuleIndex = [];
 
         var comments = this.getComments();
-        var commentRe = /(jscs\s*:\s*(en|dis)able)(.*)/;
+        var blockRe = /(jscs\s*:\s*(en|dis)able)(.*)/;
+        var lineRe = /(jscs\s*:\s*ignore)(\s+\w.*)/;
```

Comment:
Well they're not really- they could just be literals within the loop. I don't know why the original author decided to make it a variable.

Would you prefer them not to be variables?